### DISCUSSION TITLE
Support for `nonce` attribute for inline style tags

### DISCUSSION THREAD

**Discussion by bcalou** [gpbl/react-day-picker#1758] [Ideas]
Hi and thanks for this great library :)
Would you welcome a PR adding support for the nonce attribute ?
This would allow to use Content-Security-Policy headers (see https://content-security-policy.com/unsafe-inline/). Currently we have to add the `unsafe-inline` header, which is considered, well, unsafe ;)

Something like that:

```tsx
<DayPicker
    nonce="..."
```

The nonce value would then be added to the inline `style` tag.

**Comment by gpbl** [comment]
Thanks, working on this in #1834.
